main.py

```
@@ -27,74 +31,53 @@
 logger = logging.getLogger("weather-revenue-tracker")
 
+# ---------------------------------------------------------------------------
+# Signal registry — add new signals here
+# ---------------------------------------------------------------------------
+
+SIGNALS: list[BaseSignal] = [
+    WeatherSignal(),
+]
+
 
 def run_once(dry_run: bool = False) -> bool:
```

Ruff (0.15.2)

[error] Boolean-typed positional argument in function definition

(FBT001)